backend/inspirehep/hal/core/sword.py

```diff
@@ -92,4 +92,4 @@ def _create_payload(tei, doc_file):
 
         return temp_file, "application/zip", "meta.xml"
 
-    return tei, "text/xml", "meta.xml"
+    return tei.encode("utf-8"), "text/xml", "meta.xml"
```

Comment:
could we add a test for that?

Reply:
@oguzdemirbasci  :)

Reply:
Tested with a record that had given the encoding error on QA before. VCR casette generated.